web/src/components/people/AdminView.tsx

```diff
@@ -39,42 +44,93 @@ export default function AdminView() {
     fetchPeople();
   }, []);
 
+  const updatePersonUI = (id: string, training?: TrainingRecord) => {
+    // get people object and find the person with the right id
```

Comment:
How come the `TrainingRecord` is optional? What is the scenario where this would not be passed in?